client/my-sites/site-monitoring/logs-tab.tsx

```diff
@@ -116,7 +116,7 @@ export const LogsTab = ( {
 	};
 
 	return (
-		<>
+		<div class="site-logs-container">
```

Comment:
JXS is weird 😅 It's got to be `className`

```suggestion
		<div className="site-logs-container">
```

Reply:
Turns out this is the source of the CI unit tests failing.

Reply:
How I miss that? :facepalm: Thanks for pointing this out!